### PR TITLE
SOLR-16314: fix warnings about method visibility

### PR DESCRIPTION
**Pull request by epugh**
https://issues.apache.org/jira/browse/SOLR-16314

# Description

IntelliJ flagged we use private methods with our `@Before` etc annotations.   They do actually get called however. 
# Solution

We have a mix of public and private, so lets just use public.

# Tests
Ran tests

# Checklist

Please review the following and check all that apply:

- [ ] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
- [ ] I have created a Jira issue and added the issue ID to my pull request title.
- [ ] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [ ] I have developed this patch against the `main` branch.
- [ ] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

File: solr/core/src/test/org/apache/solr/search/facet/TestCloudJSONFacetJoinDomain.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public class TestCloudJSONFacetJoinDomain extends SolrCloudTestCase {
   private static final ArrayList<HttpSolrClient> CLIENTS = new ArrayList<>(5);
 
   @BeforeClass
-  private static void createMiniSolrCloudCluster() throws Exception {
+  public static void createMiniSolrCloudCluster() throws Exception {
     // sanity check constants
```
Comment (cpoerschke) [inline solr/core/src/test/org/apache/solr/search/facet/TestCloudJSONFacetJoinDomain.java:92]:
```suggestion
    // check constants
```

File: solr/core/src/test/org/apache/solr/search/facet/TestCloudJSONFacetSKG.java [hunk 2]
```diff
@@ -114,7 +114,7 @@ public class TestCloudJSONFacetSKG extends SolrCloudTestCase {
   private static final ArrayList<HttpSolrClient> CLIENTS = new ArrayList<>(5);
 
   @BeforeClass
-  private static void createMiniSolrCloudCluster() throws Exception {
+  public static void createMiniSolrCloudCluster() throws Exception {
     // sanity check constants
```
Comment (cpoerschke) [inline solr/core/src/test/org/apache/solr/search/facet/TestCloudJSONFacetSKG.java:118]:
```suggestion
    // check constants
```

File: solr/core/src/test/org/apache/solr/search/facet/TestCloudJSONFacetSKGEquiv.java [hunk 3]
```diff
@@ -103,7 +103,7 @@ public class TestCloudJSONFacetSKGEquiv extends SolrCloudTestCase {
   private static final ArrayList<HttpSolrClient> CLIENTS = new ArrayList<>(5);
 
   @BeforeClass
-  private static void createMiniSolrCloudCluster() throws Exception {
+  public static void createMiniSolrCloudCluster() throws Exception {
     // sanity check constants
```
Comment (cpoerschke) [inline solr/core/src/test/org/apache/solr/search/facet/TestCloudJSONFacetSKGEquiv.java:107]:
```suggestion
    // check constants
```

**Review by cpoerschke — APPROVED** [review]

**Review by cpoerschke — APPROVED** [review]

**Comment by epugh** [comment]
I don't want to lose the work, so I'm retesting it now and will commit it.  And per the discussion on https://lists.apache.org/thread/5k1x8rb67gh2f2kcjqwk3hbbprq9xt2r, will move to fixing the tests package by package instead of "kind of warning by kind of warning" ;-)

**Pull request merged**